assets/app/views/_alerts.html

```diff
@@ -10,7 +10,7 @@
     </div>  
     <div ng-switch-when="success" class="alert alert-success">
       <span class="pficon pficon-ok"></span>
-      <strong ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</strong><span ng-if="alert.details">{{alert.details}}</span>
+      <strong ng-if="alert.message" style="margin-right: 5px;">{{alert.message}}</strong><span ng-if="alert.details">{{alert.details}}</span><a ng-if="alert.link" href="{{alert.link}}">Link</a>
```

Comment:
saying "Link" for the text doesnt look good.  you need to add alert.linkText, or do a subobject alert.link = {url: url, text: text}, we probably eventually need to support an onclick instead of a url but we can do that later.

also refer to the patternfly reference markup for alerts, the link needs the class alert-link  https://www.patternfly.org/widgets/#alerts